DQM/Integration/python/clients/castor_dqm_sourceclient-live_cfg.py

```diff
@@ -156,85 +156,8 @@
-###                           dump2database       = cms.untracked.bool(False),
-###                           pedestalsInFC = cms.untracked.bool(False),			   
-###		 
-###			   # Define Labels
-###                           digiLabel            = cms.InputTag("castorDigis"),
-###                           rawLabel             = cms.InputTag("source"),
-###                           unpackerReportLabel  = cms.InputTag("castorDigis"),
-###                           CastorRecHitLabel    = cms.InputTag("castorreco"),
-###                           CastorTowerLabel     = cms.InputTag("CastorTowerReco"),
-###                           CastorBasicJetsLabel = cms.InputTag("ak7BasicJets"),
-###                           CastorJetIDLabel     = cms.InputTag("ak7CastorJetID"),
-###                          
-###			   DataIntMonitor= cms.untracked.bool(True),
-###			   TowerJetMonitor= cms.untracked.bool(False),
-###
-###                           DigiMonitor = cms.untracked.bool(True),
-###                           DigiPerChannel = cms.untracked.bool(True), 
-###                           DigiInFC = cms.untracked.bool(False),
-###                          
-###                           RecHitMonitor = cms.untracked.bool(True), 
-###			   RecHitsPerChannel = cms.untracked.bool(True),
-###
-###                           ChannelQualityMonitor= cms.untracked.bool(True),
-###                           nThreshold = cms.untracked.double(60),
-###                           dThreshold = cms.untracked.double(1.0),
-###                           OfflineMode = cms.untracked.bool(False),
-###                           averageEnergyMethod = cms.untracked.bool(True),          
-###
-###                           PSMonitor= cms.untracked.bool(True),
-###                           numberSigma = cms.untracked.double(1.5),
-###                           thirdRegionThreshold =  cms.untracked.double(999999.0), # to avoid empty cells in reportSummary            
-###                           EDMonitor= cms.untracked.bool(True),
-###                           HIMonitor= cms.untracked.bool(True),
-###                                       
-###                           diagnosticPrescaleTime = cms.untracked.int32(-1),
-###                           diagnosticPrescaleUpdate = cms.untracked.int32(-1),
-###                           diagnosticPrescaleLS = cms.untracked.int32(-1),
-###                             
-###                           LEDMonitor = cms.untracked.bool(True),
-###                           LEDPerChannel = cms.untracked.bool(True),
-###                           FirstSignalBin = cms.untracked.int32(0),
-###                           LastSignalBin = cms.untracked.int32(9),
-###                           LED_ADC_Thresh = cms.untracked.double(-1000.0)      
-###                           )
+from EventFilter.CastorRawToDigi.CastorRawToDigi_cff import *
+process.castorDigis = castorDigis.clone( UnpackZDC = cms.bool(False) )
```

Comment:
I mean, ZDC DQM is in the hcal_dqm_sourceclient-live_cfg.py.
In castor_dqm_sourceclient-live_cfg.py, there is no need to unpack ZDC, anyway, if you want to put True, it's also fine.